nd4j/nd4j-parameter-server-parent/nd4j-parameter-server-node/src/main/java/org/nd4j/parameterserver/distributed/v2/chunks/impl/InmemoryChunksTracker.java

```diff
@@ -0,0 +1,121 @@
+/*******************************************************************************
+ * Copyright (c) 2015-2018 Skymind, Inc.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Apache License, Version 2.0 which is available at
+ * https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ******************************************************************************/
+
+package org.nd4j.parameterserver.distributed.v2.chunks.impl;
+
+import lombok.Getter;
+import lombok.NonNull;
+import lombok.extern.slf4j.Slf4j;
+import lombok.val;
+import org.nd4j.linalg.exception.ND4JIllegalStateException;
+import org.nd4j.linalg.primitives.AtomicBoolean;
+import org.nd4j.linalg.util.SerializationUtils;
+import org.nd4j.parameterserver.distributed.v2.chunks.ChunksTracker;
+import org.nd4j.parameterserver.distributed.v2.chunks.VoidChunk;
+import org.nd4j.parameterserver.distributed.v2.messages.VoidMessage;
+
+import java.io.*;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+
+/**
+ * Memory-based implementation of ChunksTracker
+ */
+@Slf4j
+public class InmemoryChunksTracker<T extends VoidMessage> implements ChunksTracker<T> {
```

Comment:
"Inmemory" -> "InMemory"

Reply:
Huh, it's not that trivial change in git on Windows :)